.bit/responses/3.1-Week 3 Step 1.md

```diff
@@ -74,8 +74,10 @@ A **storage account** provides a unique namespace in Azure for your data. **Ever
 <br><br/>
 </details>
 
+> ⚠️ **Warning:** The set-up for your blob storage container makes the endpoint **publicly accessible.** This means that it is *not* safe for real-world applications since it allows the potential for anyone with the endpoint to access different blobs they should not have access to.
```

Comment:
To be consistent we should use Blob with capital "B" if we are addressing the Azure Service in the document